src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -155,18 +156,39 @@ func Update(mutate func(cfg *Config) error) error {
 // InConfigDir returns the path to the given filename inside of the configdir.
 func InConfigDir(filename string) (string, error) {
 	cdir := *configdir
+
 	if cdir == "" {
-		cdir = appdir.General("Lantern")
+		if runtime.GOOS == "linux" {
+			// It is more common on Linux to expect application related directories
+			// in all lowercase. The lantern wrapper also expects a lowercased
+			// directory.
+			cdir = appdir.General("lantern")
+
+			// Backwards compatibility fix.
+			oldDir := appdir.General("Lantern")
```

Comment:
I actually don't think we should worry about backwards compatibility for this at this point. This has still only been released in beta, and there are almost no user-configurable settings. So I would skip adding this complexity at this point.